packages/workflows/src/Create.ts

```diff
@@ -72,6 +72,7 @@ export class CreateWorkflow{
     public static createWorkflow(session: AbstractSession, WorkflowName: string, WorkflowDefinitionFile: string,
                                  systemName: string, Owner: string, VariableInputFile?: string, Variables?: string,
                                  AssignToOwner?: boolean, AccessType?: accessT, DeleteCompletedJobs?: boolean,
+                                 JobStatement?: Array<string>,
                                  zOSMFVersion = WorkflowConstants.ZOSMF_VERSION
                                     // add job statement, account info, comments and resolveGlobalConflictByUsing,
                                     ): Promise<ICreatedWorkflow> {
```

Comment:
I'm afraid this may be a breaking change : (
Someone may already be expecting zosmfVersion to be the 11th parameter on the `createWorkflow`  API.

Reply:
Hard to say. Technically I could keep the zosmfVersion as the 10th parameter, but then it would be better to switch it from WorkflowConstants.ZOSMF_VERSION to Null and just replace it in the code with the constant, else I would have to import the WorkflowConstants everywhere where the job statement was added.

I did it this way because that parameter is barely used in the code and unfortunately Typescript doesn't allow to skip random parameters, only the last ones.

Reply:
Hey @abastyr,
Gotcha. I definitely see the benefit in having the version as the last parameter since it can be skipped (and a value will still be available for it.

However, say that someone is calling this function like `createWorkflow(session, "name", "defFile","SYS","myID", null, null, null, null, false, "1.0")`

Once they update their CLI version to `^6.x.x`, they may start getting compilation errors since they didn't provide an array in the place they should have. And even if they don't get it at compile/transpile -time, `"1.0".join("\n")` should complain since strings do not know about `.join`.

We can still do the breaking change, but this PR will have to go out as part of the `next` release.

@dkelosky, thoughts?

Reply:
I think I like breaking the API for `next` and passing an interface of options going forward; otherwise, that's a lot of `null`s to keep straight 😄